### PR TITLE
Fixes alignment bugs in media wall design

### LINKED ISSUE #683: UI bugs in Media Wall Design
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
In the Media Wall Design section in the live feed section, there are some UI bugs like:
1. Alignment bugs of the DD menu
2. The corresponding field is going in two lines
3. The font size of the title is less than the subtitle.

**Expected behavior:**
These bugs need to be fixed

**Screenshots of the issue:**
screen shot 2018-03-15 at 11 14 38 pm

I would like to work on this issue 😄

### PR DESCRIPTION
**Pull request by Remorax**
**Changes proposed in this pull request**
- Fixed the alignment-bugs
- Moved the apply button a bit lower so the dropdown menu does not overlap with the button
- Increased the size of the heading as compared to the subheadings

**Screenshots (if appropriate)** 
screen shot 2018-03-16 at 2 10 43 am

Link: [https://pr-694-fossasia-loklaksearch.surge.sh](https://pr-694-fossasia-loklaksearch.surge.sh)

Closes #683

### CODE REVIEW

File: src/app/media-wall/media-wall-design/media-wall-design.component.scss [hunk 1]
```diff
@@ -87,3 +87,31 @@ span.apply-button {
...
+}
+
+#count1 {
+	margin-right: 20px;
+	width: 40%;
+}
+
+#card_style {
+	 margin-top: 12px;
+}
```
Comment (ParthS007) [inline src/app/media-wall/media-wall-design/media-wall-design.component.scss]: Please add a line at EOF :+1:

File: src/app/media-wall/media-wall-design/media-wall-design.component.html [hunk 2]
```diff
@@ -71,4 +74,4 @@ <h3>Cards Design</h3>
 			</button>
 		</span>
 	</div>
-</div>
+</div>
```
Comment (simsausaurabh) [inline src/app/media-wall/media-wall-design/media-wall-design.component.html]: Add a new line at the end.

**Comment by Remorax** [comment]
@simsausaurabh As regards the link, I think there is a problem in viewing the media wall when the project is deployed to surge, we've been through this before and that's why I've provided screenshots for the change. I guess you could also pull my fork and check it once, that would be great! But this surge error needs to be fixed as part of a different issue 😄

File: src/app/media-wall/media-wall-design/media-wall-design.component.scss (hunk 1 above)
Comment (Remorax) [reply src/app/media-wall/media-wall-design/media-wall-design.component.scss]: Done!

File: src/app/media-wall/media-wall-design/media-wall-design.component.html (hunk 2 above)
Comment (Remorax) [reply src/app/media-wall/media-wall-design/media-wall-design.component.html]: Done!

**Comment by singhpratyush** [comment]
@Remorax: Thanks for the fix.

**Pull request merged**